app/assets/stylesheets/application.scss

```diff
@@ -13,9 +13,28 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
-
  */
 .cancel {
   text-align: center;
+}
+
+body {
+  background-image: image_url("");
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores